app/assets/stylesheets/ppm-estimator.print.scss

```diff
@@ -0,0 +1,15 @@
+@import "application.scss";
```

Comment:
This file does not currently live in the `components/` subdirectory which would be its more natural home, because it needs to import `application.scss`. IIRC, using `..` in the import declaration did not work.

Reply:
You shouldn't have to include the entirety of `application.scss` in the print stylesheet. Styles from `application.scss` are applied across all mediums (e.g. `screen`, `print`, etc.) so you should be able to rely on the cascade here and remove this `@import`.

Reply:
To clarify my previous comment, looking at the `<head>`:

```html
<!-- this stylesheet applies to all media types -->
<link rel="stylesheet" href="/assets/application.css">

<!-- this stylesheet applies only to print media types -->
<link rel="stylesheet" media-"print" href="/assets/print.css">
```